tools/pylint-extensions/pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -2028,6 +2028,71 @@ def _check_import(self, name, node):
                 msgid="no-raise-with-traceback", node=node, confidence=None
             )
 
+class TypePropertyNameTooLong(BaseChecker):
```

Comment:
Will this also capture if a class var or instance attr is too long? I couldn't tell from the code so asking :)

Reply:
Yes it will! I can change the checker name to make it clearer?